sources/packages/web/src/views/FinancialInfo.vue

```diff
@@ -49,6 +49,48 @@
           </div>
         </Section>
       </ContentGroup>
+      <ContentGroup>
+        <Section
+          :title="questionsFin.livingSituation.title"
+          :sub-title="questionsFin.livingSituation.subTitle"
+        >
+          <Question :text="questionsFin.livingSituation.question">
+            <RadioButtonList
+              name="livingSituation"
+              v-model="financialInfoState.livingSituationValue"
+            >
+            </RadioButtonList>
+          </Question>
+          <ContentGroup v-if="financialInfoState.livingSituationValue === 'no'">
+            <Section>
+              <Question :text="questionsFin.travelCosts.question">
+                <RadioButtonList
+                  name="travelCosts"
+                  v-model="financialInfoState.travelCostsValue"
+                >
+                </RadioButtonList>
+              </Question>
+
+              <ContentGroup
+                v-if="financialInfoState.travelCostsValue === 'yes'"
+              >
+                <Section>
+                  <Question :text="questionsFin.returnTripCosts.question">
+                    <InputNumber
+                      class="p-m-2"
+                      mode="currency"
+                      currency="CAD"
```

Comment:
Please take a look to ensure that the currency symbol is "$". It seems to be the requirement.